### PR TITLE
[Add] Gaurav Sen

### PR DESCRIPTION
**Pull request by Harsh-2909**
## Changes proposed
- Added a new subcategory for backend: system-design
- Added a new link for Gaurav Sen

### CODE REVIEW

File: database/backend/system-design.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "name": "Gaurav Sen",
+        "description": "Gaurav Sen has a youtube channel where he talks about system design concepts and interview preparation.",
```
Comment (rupali-codes) [inline database/backend/system-design.json:4]:
```suggestion
        "description": "This channel is focused on teaching System design but they also provide content related to other computer science concepts such as Algorithm design, Dynamic programming, Software engineering, Competitive programming and more.",
```

**Comment by Harsh-2909** [comment]
I have made the changes. 
But seriously, I am always bad at writing these things

File: database/backend/system-design.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "name": "Gaurav Sen",
+        "description": "This channel is focused on teaching System design but they also provide content related to other computer science concepts such as Algorithm design, Dynamic programming, Software engineering, Competitive programming and more.",
```
Comment (CBID2) [inline database/backend/system-design.json:4]:
```suggestion
        "description": "This channel teaches System Design and other computer science concepts such as Algorithm design, Dynamic programming, Software engineering, Competitive programming,and more.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hey @Harsh-2909! 😄 I just made some edits to the Description section. Other than that, great job! :smiling

**Comment by Harsh-2909** [comment]
Made the changes

**Comment by Harsh-2909** [comment]
Just an update to merge this PR as I have made the requested changes

**Comment by rupali-codes** [comment]
Thanks, looks good to me :)

**Pull request merged**